src/MediaEndPoints.cs

```diff
@@ -146,6 +148,12 @@ public enum VideoCodecsEnum
         Unknown
     }
 
+    public enum TextCodecsEnum
+    {
+        T140, //T.140 specifies that text and other T.140 elements must be transmitted in ISO 10646-1 code with UTF-8 transformation.
+        Unknown,

```

Comment:
Would it be better to put Unknown first so it becomes the default?

Reply:
Ah ok. For audio formats 0 is the well known identifier for PCMU. In the case of text codecs I suspect it will only get set in very specific cases so Unknown by default would be better.